main.js

```diff
@@ -40,6 +40,10 @@ function addToIdeaList() {
 function deleteCardEvent(event) {
   var idNum = parseInt(event.target.id);
   deleteFromArray(idNum);
+  cardGrid.innerHTML = "";
```

Comment:
We probably want to refactor this code into a function later since we're reusing it.